api/src/functionalTest/resources/application-functional-tests.properties

```diff
@@ -28,6 +33,8 @@ launchDarkly.socketTimeout=3
 launchDarkly.flushInterval=10
 pba.config1.service.names=${PBA_CONFIG1_SERVICE_NAMES:CMC}
 
+refund.api.url=${REFUND_API_URL:http://ccpay-refunds-api-pr-108.service.core-compute-preview.internal}
```

Comment:
why are we pointing to a PR here?

Reply:
Yashar this is for functional tests and its backup value for the variable refund.api.url. It will point to AAT env when it runs in pipeline.